docs/main/user-guides/observability.md

```diff
@@ -0,0 +1,273 @@
+---
+id: observability
+title: Observability
+keywords:
+ - Monitoring
+ - Observability
+ - Pool metrics
+ - Prometheus
+ - Grafana
+description: This guide explains about monitoring your cluster.
+---
+# Observability
+
+Monitoring your cluster is essential to ensure the health, efficiency, and reliability of your storage infrastructure.​ The following technologies are implemented to monitor your cluster:
```

Comment:
No comma required after efficiency?

Reply:
@abhilashshetty04 As per the Documentation style guide a "Comma" is required in a series consisting of three or more elements.